client/src/context/useKanbanContext.tsx

```diff
@@ -1,10 +1,20 @@
-import { useState, useContext, createContext, FunctionComponent, useEffect, Dispatch, SetStateAction } from 'react';
+import {
+  useState,
+  useContext,
+  createContext,
+  FunctionComponent,
+  useEffect,
+  Dispatch,
+  SetStateAction,
+  useCallback,
+} from 'react';
 import { DraggableLocation, DropResult } from 'react-beautiful-dnd';
 import cloneDeep from 'lodash.clonedeep';
```

Comment:
@moffatethan Will we still need lodash? It's an expensive library to load for a single function. Could this be done/changed?

Reply:
That's not lodash. That's lodash.clonedeep. Not a big package.

Reply:
Gotcha. I see it in now as a devdep 👌🏻